packages/@aws-cdk/aws-eks/lib/k8s-resource-handler/index.py

```diff
@@ -0,0 +1,66 @@
+import json
```

Comment:
Likewise here - would much rather have everything be `Javascript` and not sometimes Python, sometimes JS for no apparent reason.

Reply:
Same reason.